package.json

```diff
@@ -102,7 +101,7 @@
       "web/.storybook/public"
     ]
   },
-  "packageManager": "pnpm@10.18.0+sha512.e804f889f1cecc40d572db084eec3e4881739f8dec69c0ff10d2d1beff9a4e309383ba27b5b750059d7f4c149535b6cd0d2cb1ed3aeb739239a4284a68f40cfa",
+  "packageManager": "pnpm@10.20.0+sha512.cf9998222162dd85864d0a8102e7892e7ba4ceadebbf5a31f9c2fce48dfce317a9c53b9f6464d1ef9042cba2e02ae02a9f7c143a2b438cd93c91840f0192b9dd",
```

Comment:
Tag build: https://github.com/gravitational/teleport.e/actions/runs/19066222756

Reply:
It compiles on all three platforms and the app works fine on macOS. I couldn't test it on Windows as the app is broken on master atm (#61044).